modules/cluster-cloud-controller-config-osp.adoc

```diff
@@ -0,0 +1,296 @@
+
+//  To override this behavior, specify a comma-separated list of CIDR addresses. CCM sorts and groups all addresses that match the list in a prioritized manner, wherein the first retrieved item has  a higher priority than the last. Addresses that do not match the list remain in their default order. The default value is `""`.
+
+// This option can be useful if you have multiple or dual-stack interfaces attached to a node that need a user-controlled, deterministic way of sorting addresses.
+// |===
+
+[id="ccm-config-lb-options"]
+== Load balancer options
+
+CCM supports several load balancer options for deployments that use Octavia. 
+
+[NOTE]
+====
+Neutron-LBaaS support is deprecated. 
+====
+
+// Commented options are untested as of yet.
+
+|===
+| Option | Description
+
+| `enabled`
+| Whether or not to enable the `LoadBalancer` type of services integration. The default value is `true`.
+
+// Removed: always enforced.
+//| `use-octavia`
+//| Whether or not to use Octavia for the `LoadBalancer` type of service implementation rather than Neutron-LBaaS. The default value is `true`.
+
+| `floating-network-id`
+| Optional. The external network used to create floating IP addresses for load balancer virtual IP addresses (VIPs). If there are multiple external networks in the cloud, this option must be set or the user must specify `loadbalancer.openstack.org/floating-network-id` in the service annotation.
+
+| `floating-subnet-id`
+| Optional. The external network subnet used to create floating IP addresses for the load balancer VIP. Can be overridden by the service annotation `loadbalancer.openstack.org/floating-subnet-id`.
+
+| `floating-subnet`
+| Optional. A name pattern (glob or regular expression if starting with `~`) for the external network subnet used to create floating IP addresses for the load balancer VIP. Can be overridden by the service annotation `loadbalancer.openstack.org/floating-subnet`. If multiple subnets match the pattern, the first one with available IP addresses is used.
+
+| `floating-subnet-tags`
+| Optional. Tags for the external network subnet used to create floating IP addresses for the load balancer VIP. Can be overridden by the service annotation `loadbalancer.openstack.org/floating-subnet-tags`. If multiple subnets match these tags, the first one with available IP addresses is used.
+
+If the {rh-openstack} network is configured with sharing disabled, for example, with the `--no-share` flag used during creation, this option is unsupported. Set the network to share to use this option.
+
+| `lb-method`
+| The load balancing algorithm used to create the load balancer pool. The value can be `ROUND_ROBIN`, `LEAST_CONNECTIONS`, or `SOURCE_IP`. The default value is `ROUND_ROBIN`.
+
+For the OVN provider, only the `SOURCE_IP` algorithm is supported.
```

Comment:
The algorithm for the OVN octavia provider is SOURCE_IP_PORT instead of SOURCE_IP.
For reference: https://access.redhat.com/documentation/fr-fr/red_hat_openstack_platform/16.2/html/using_octavia_for_load_balancing-as-a-service/create-other-kinds-lbs_rhosp-lbaas#create-ovn-lb_create-other-kinds-lbs
```suggestion
For the OVN provider, only the `SOURCE_IP_PORT` algorithm is supported.
```